README.md

```diff
@@ -34,31 +34,38 @@ Symfony Bundle to log Request/Response with Monolog.
 
 ## Usage
 
-**```config.yml```** :
+```yaml
+# app/config.yml
 
+m6_web_log_bridge:
+    resources:
+        - %kernel.root_dir%/config/m6web_log_bridge.yml
+    content_formatter: m6web_log_bridge.log_content_provider # Provider service name
+    ignore_headers: # key list from mask/ignore header info
+        - php-auth-pw
+    prefix_key: ~ # define prefix key on log context
 ```
-    #app/config.yml
-    m6_web_log_bridge:
-        logger: monolog.service_name
-        resources:
-            - %kernel.root_dir%/config/m6web_log_bridge.yml
-        content_formatter: m6web_log_bridge.log_content_provider # Provider service name
-        ignore_headers: # key list from mask/ignore header info
-            - php-auth-pw
-        prefix_key: ~ # define prefix key on log context
-```
 
-You can use the Logger provided by the bundle
+This bundle provide an default logger with monolog support `m6web_log_bridge.logger`
+
+You can overwrite this logger :
```

Comment:
By default, this bundle use a builtin logger with monolog support `m6web_log_bridge.logger`
You can override this configuration by writing your own logger who must implements `Psr\Log\LoggerInterface`